src/handleAction.js

```diff
@@ -5,9 +5,13 @@ function isFunction(val) {
 }
 
 export default function handleAction(type, reducers) {
+  const typeValue = isFunction(type)
```

Comment:
Do you mean `const typeValue = String(type)` instead of `const typeValue = isFunction(type) ? type.toString() : type` ?